pgbouncer/metadata.csv

```diff
@@ -29,3 +29,9 @@ pgbouncer.pools.maxwait_us,gauge,,microsecond,,Microsecond part of the maximum w
 pgbouncer.databases.pool_size,gauge,,connection,,Maximum number of server connections,0,pgbouncer,database pool size,
 pgbouncer.databases.max_connections,gauge,,connection,,Maximum number of allowed connections,0,pgbouncer,database max conns,
 pgbouncer.databases.current_connections,gauge,,connection,,Current number of connections for this database,0,pgbouncer,database current conns,cpu
+pgbouncer.clients.connect_time,gauge,,time,,"When the connection was made",-1,pgbouncer,client connect time,
+pgbouncer.clients.request_time,gauge,,time,,"When last request was issued",-1,pgbouncer,client request time,
+pgbouncer.clients.wait,gauge,,second,,"Current waiting time in seconds",-1,pgbouncer,client wait,
+pgbouncer.clients.wait_us,gauge,,microsecond,,Microsecond part of the current waiting time,-1,pgbouncer,client wait,
+pgbouncer.servers.connect_time,gauge,,time,,"When the connection was made",-1,pgbouncer,server connect time,
+pgbouncer.servers.request_time,gauge,,time,,"When last request was issued",-1,pgbouncer,server request time,
```

Comment:
Since the `connect_time` and `request_time` metrics are generated from `time.mktime()`, should we indicate in the metric descriptions that these metrics refer to the time in seconds since epoch time? Perhaps we could also update the unit name for those metrics to `second`. 

Also, can we rephrase "Microsecond part of the current waiting time" to "The microsecond portion of the current waiting time"

Reply:
Done.

Reply:
Thanks for making the change! What do you think about my note regarding clarifying the metric descriptions and/or unit for the `.connect_time` and `request_time` metrics to indicate that the values are in `seconds` since unix epoch time?

Reply:
I thought I'd added that, but just realized I'd only added to some but not all of the fields. Also I somehow missed your comment about updating the unit name to `second`. I've pushed updates for both of those oversights.